app/authentication.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from flask import current_app, abort, request
+
+
+def requires_authentication():
```

Comment:
Note to self; this module is another thing that might be a candidate for moving out into a shared library.